QSB/TimeSync/WakeUpSync.cs

```diff
@@ -165,19 +147,31 @@ private void StartPausing()
             {
                 return;
             }
-            OWTime.Pause(OWTime.PauseType.Menu);
-            Time.timeScale = 0f;
+            _timeScale = 0f;
             _state = State.Pausing;
         }
 
-        private void StopPausing()
+        private void ResetTimeScale()
         {
-            if (_state != State.Pausing)
+            _timeScale = 1f;
+            _state = State.Awake;
+
+            if (_isInputDisabled)
             {
-                return;
+                EnableInput();
             }
-            OWTime.Unpause(OWTime.PauseType.Menu);
-            _state = State.Awake;
+        }
+
+        private void DisableInput()
+        {
+            _isInputDisabled = true;
```

Comment:
Can you invert this and call it enabled instead of disabled?

Reply:
Done